.gitignore

```diff
@@ -11,7 +11,6 @@
 .mtj.tmp/
 
 # Package Files #
-*.jar
```

Comment:
jar is removed?

Reply:
Need to add a jar file for integration test.

Reply:
how about use /test/function/xxx/*.jar to make it explicit

Reply:
done